hyperspy/io_plugins/hdf5.py

```diff
@@ -425,9 +425,9 @@ def parse_structure(key, group, value, _type, compression):
             try:
                 group.attrs[key] = value
             except:
-                print("The hdf5 writer could not write the following "
-                      "information in the file")
-                print('%s : %s' % (key, value))
+                warnings.warn(
```

Comment:
I would use `logger` here - you want to print all the things that were not saved, all the time. Currently it only prints the warnings for the first file created.